### ISSUE TITLE
Another container besides AWS?

### ISSUE THREAD

**Issue by MB34** [AnomalyInnovations/serverless-stack-com#174]
Do you have experience doing this in another container such as Docker or Heroku?

**Comment by jayair** [comment]
@MB34 Hmm I'm not sure what you mean by container here. There are open source implementations of Lambda but I have not tried them yet.

**Comment by Snake-Tn** [comment]
I guess he meant testing the stack locally using docker containers to mimic AWS environment. 
As deploying on every change makes the development feedback loop slow.

**Comment by jayair** [comment]
Yeah the local development workflow is tricky to get up and running. One of our readers tried something - https://discourse.serverless-stack.com/t/best-local-development-workflow-for-the-tutorial/731/2?u=jayair

**Comment by jkketts** [comment]
Would we be able to use something like https://github.com/localstack/localstack for this? That might be a better approach than trying to figure out all the complexities involved with containers.
